modules/local/exomedepth/merge_cnv/main.nf

```diff
@@ -0,0 +1,25 @@
+//MERGE CNV CALL FILES
+process CNV_MERGE {
+    tag "$meta"
+
+    publishDir "$params.outdir/exomedepth/cnv_call", mode: 'copy'
+
+    input:
+    tuple val(meta), path(auto), path(chrx)
+
+    output:
+    tuple val(meta), path("*.txt")
+
+    script:
+    prefix = task.ext.prefix ?: "${meta}_CNVs_ExomeDepth"
```

Comment:
```suggestion
    def prefix = task.ext.prefix ?: "${meta}_CNVs_ExomeDepth"
```

This makes sure this variable can only be accessed in this `script` block

Reply:
Fixed